portfolio/src/main/webapp/script.js

```diff
@@ -93,4 +102,23 @@ function createCommentDiv(commentObj) {
   // TODO: Convert from fairly useless MS time to hour/minute.
 
   return commentDiv;
-}
\ No newline at end of file
+}
+
+function loadMap() {
+  let script = document.createElement('script');
+  script.src = `https://maps.googleapis.com/maps/api/js?key=AIzaSyAqY8BqWGKw-Nr
+      wfYOeoOATcbUZkQV1R2w&callback=initMap`;
+  script.defer = true;
+  script.async = true;
+
+  // Attach the callback function to the `window` object
+  // Executes once the API is loaded and available.
+  window.initMap = function() {
+    map = new google.maps.Map(document.getElementById('map'), {
+    center: {lat: 40.730610, lng: -73.935242},
```

Comment:
Add an indent for 'center' and 'zoom'

Reply:
This actually ended up being included in #19 ? I hope that's OK!

Reply:
no worries